build.sbt

```diff
@@ -59,23 +60,13 @@ lazy val root = (project in file("."))
       "org.json4s" %% "json4s-jackson" % "4.0.6",
       
       "jakarta.ws.rs" % "jakarta.ws.rs-api" % "[3.1.0]",
-      // "org.glassfish.jersey.core" % "jersey-common" % "1.2.1",             // Required at runtime by javax.ws.rs-api
       "com.github.swagger-akka-http" %% "swagger-pekko-http" % "[2.14.0]",
-      "com.github.swagger-akka-http" %% "swagger-scala-module" % "[2.12.0,)",
-      //"io.swagger.core.v3" % "swagger-core-jakarta" % "[2.1.12]",             // Version 2.1.13+ requires newer versions of slick and slick-hikaricp
-      //"io.swagger.core.v3" % "swagger-jaxrs2-jakarta" % "[2.1.12]",           // Version 2.1.13+ requires newer versions of slick and slick-hikaricp
       
       "ch.qos.logback" % "logback-classic" % "1.5.6",
-      //"net.logstash.logback" % "logstash-logback-encoder" % "[7.4,)",
-      // "com.typesafe.slick" %% "slick" % "[3.3.3]",             // Version 3.4.1 depends on slick-pg and slick-pg_json4s v0.21.0
       "com.typesafe.slick" %% "slick-hikaricp" % "[3.4.1]",       // Version 3.4.1 depends on slick-pg and slick-pg_json4s v0.21.0
       // "com.github.tminglei" %% "slick-pg" % "[0.20.4]",        // Version 0.21.0 depends on version 3.4.0 of slick and slick-hikaricp
```

Comment:
Do we need to delete thi?

Reply:
Great catch. No we do not. slick-pg gets brought into the dependency stack transitively via slick-pg_json4s. Removing this line.